Sorts/test/SimplifiedWiggleSort.test.js

```diff
@@ -0,0 +1,29 @@
+import { simplifiedWiggleSort } from '../SimplifiedWiggleSort.js'
+
+describe('simplified wiggle sort', () => {
+  test('given array of symbols [\'b\', \'a\', \'c\'] return wiggle sorted array [\'a\', \'c\', \'b\']', () => {
+    const src = ['a', 'b', 'c']
+    expect(simplifiedWiggleSort(src)).toEqual(['a', 'c', 'b'])
+  })
+
+  test('wiggle sort with duplicates, even array', () => {
+    const src = [2, 2, 1, 3]
+    expect(simplifiedWiggleSort(src)).toEqual([1, 3, 2, 2])
+  })
+
+  test('wiggle sort with duplicates, odd array', () => {
+    const src = [1, 1, 1, 2, 4]
+    expect(simplifiedWiggleSort(src)).toEqual([1, 4, 1, 2, 1])
+  })
+
+  test('simplified wiggle sort for [3, 5, 6, 1]', () => {
```

Comment:
Again, please don't put the data in the test message.

Reply:
Sorry, I didn't realise that's a problem. I will fix this.